src/main/java/com/TableTalk/Enterprise/dto/User.java

```diff
@@ -1,26 +1,18 @@
 package com.TableTalk.Enterprise.dto;
 
-
-/*
-This could all be completely wrong considering I'm unfamiliar with firebase.
- */
-
```

Comment:
Same thing here, no commented out code.

But also, its OK, we're all learning here.